mne/io/__init__.py

```diff
@@ -39,8 +39,8 @@
 from .eeglab import read_raw_eeglab, read_epochs_eeglab
 
 # for backward compatibility
-from .fiff import RawFIF
-from .fiff import RawFIF as Raw
+from .fiff import Raw
+from .fiff import Raw as RawFIF
```

Comment:
In master we mostly use Raw not RawFIF. This just pushes is more in that
direction. WDYT?